packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionHistory.tsx

```diff
@@ -25,7 +25,7 @@ export const TransactionHistory = () => {
     data: { transactions },
     loading,
     completed,
-    paused,
+    resume,
```

Comment:
i don't think the resume state should be shared

because i am at the bottom of the pending txs table, but not the settled txs table (i didn't check that tab)

and clicking "load more" on the pending tab would not load more because i haven't scrolled to the bottom of the settled txs table

<img width="987" alt="image" src="https://github.com/OffchainLabs/arbitrum-token-bridge/assets/13184582/dc6de4fb-7cef-45a0-9c50-c7b44508ce0b">

Reply:
Discussed during OH